### ISSUE TITLE
Question regarding DLC

### ISSUE THREAD

**Issue by gvaldebenit** [ScarletStudy/DGS1-3DS-Release#62]
Hi, first I would like to thank you for all you hard efforts translating this. You made great work on caring about everything a professional translation team would do.
So, I decided to wait until Ch. 5 translation was ready to install it, but after all process, the DLC option is missing, so I started a new game (in case it appear later when you have a save) but nothing.
It is intentional?
If so, there's a chance to give support to it? (like translating?)
Thanks again for the Localization of DGS and I hope the DGS2 as well.

**Comment by wabberz** [comment]
Yes, we've removed the DLC button for now. It won't work anyways without the right locale for Luma3DS and there is only one complicated way for O3DS to acquire it legally - something I'm not sure many of our users will be able to do.

The DGS1 DLC is mostly concept art, beta music, developer videos and short skids (not real cases, just the characters talking about something for a short while - probably not canon). It's "nice to have", but not really vital, which is why DGS2 has the priority. We'll think about if and how we would do DLC, especially since the DGS2 DLC is actually interesting and worth it - both are unfortunately unavailable on Android though.

**Comment by gvaldebenit** [comment]
Yeah, I understand, it's unfortunate that dlc is not avaliable on Android, and more if DGS2 dlc is really worth it. Keep it up and thanks for the reply.
